src/app/helptext/system/ca.ts

```diff
@@ -24,6 +24,23 @@ export const helptext_system_ca = {
       validation: [Validators.required]
     },
 
+    key_type: {
+      placeholder: T("Key Type"),
+      tooltip: T(
+        'See <a href="https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa"></a>\
```

Comment:
These links should have link text:
```suggestion
        'See <a href="https://crypto.stackexchange.com/questions/1190/why-is-elliptic-curve-cryptography-not-widely-used-compared-to-rsa">Why is elliptic curve cryptography not widely used, compared to RSA?</a>\
```